### ISSUE TITLE
No option to "Apply Theme" and "Apply tweaks" error

### ISSUE THREAD

**Issue by mdolce8** [mallowigi/mtslack#57]
Hi, I just installed `mtslack` following the README. It appears successful, but when doing `sudo mtslack`, I don't see an option for "Apply Theme" or "Remove Theme". Instead, I see "Apply tweaks, Restore slack, Display mtslack Version, and Exit". And when I tried to apply tweaks I get the following error below. I have followed the instructions as they are written, is there a fix to this? Thanks  

I am on: macOS 10.15.7
Slack 4.12.0
mtslack 16.1.0 

```? Please select an command Apply tweaks

Adding Tweaks Code to Slack... 

node:fs:2045
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, copyfile '/Applications/Slack.app/Contents/Resources/app-x64.asar' -> '/Applications/Slack.app/Contents/Resources/app-x64.asar.backup'
    at Object.copyFileSync (node:fs:2045:3)
    at Object.exports.apply (/usr/local/lib/node_modules/mtslack/lib/command.js:174:6)
    at Object.exports.selectTheme (/usr/local/lib/node_modules/mtslack/lib/command.js:212:8)
    at Object.exports.change (/usr/local/lib/node_modules/mtslack/lib/command.js:203:8)
    at exports.execute (/usr/local/lib/node_modules/mtslack/lib/command.js:218:12)
    at run (/usr/local/lib/node_modules/mtslack/main.js:32:3)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
    at async AutoUpdate.<anonymous> (/usr/local/lib/node_modules/mtslack/main.js:60:7) {
  errno: -2,
  syscall: 'copyfile',
  code: 'ENOENT',
  path: '/Applications/Slack.app/Contents/Resources/app-x64.asar',
  dest: '/Applications/Slack.app/Contents/Resources/app-x64.asar.backup'
}
```

**Comment by mallowigi** [comment]
It's just a wording, I've updated the menu because it doesn't just apply a theme but add the Slack Tweaks as well.

I need to update the doc.

**Comment by mdolce8** [comment]
Hi, I tried again after re-installing slack and see the same error as my previous post. Looking at the first line, I do not have `'/Applications/Slack.app/Contents/Resources/app-x64.asar' -> '/Applications/Slack.app/Contents/Resources/app-x64.asar.backup'` , but I do have `/Applications/Slack.app/Contents/Resources/app.asar` , so it seems the "x64" file is missing?

**Comment by mallowigi** [comment]
This is available since the latest version of Slack.

Please update Slack, currently there's no way of handling both cases.

Elior Boukhobza

Programmer, Designer and Food Lover ;)

Mallowigi
heliosaian@gmail.com
https://www.material-theme.com
[image: twitter] <https://twitter.com/Mallowigi>
[image: linkedin] <https://www.linkedin.com/in/eliorboukhobza/>
[image: instagram] <https://www.instagram.com/heliosaian/>
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

On 5 January 2021 at 15:26:52, Michael Dolce (notifications@github.com)
wrote:

Hi, I tried again after re-installing slack and see the same error as my
previous post. Looking at the first line, I do not have
'/Applications/Slack.app/Contents/Resources/app-x64.asar'
-> '/Applications/Slack.app/Contents/Resources/app-x64.asar.backup' , but I
do have /Applications/Slack.app/Contents/Resources/app.asar , so it seems
the "x64" file is missing?

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/mallowigi/mtslack/issues/57#issuecomment-754635117>, or
unsubscribe

**Comment by mdolce8** [comment]
Yes, I have the latest version of Slack already (4.12.0). And I have reinstalled it as you suggested. I am still seeing the same error

**Comment by mallowigi** [comment]
So that means perhaps that you don't have a 64bit computer?

Anyway I'll try to figure out.

Try to connect to our Slack channel so we can discuss it together.

Elior Boukhobza

Programmer, Designer and Food Lover ;)

Mallowigi
heliosaian@gmail.com
https://www.material-theme.com
[image: twitter] <https://twitter.com/Mallowigi>
[image: linkedin] <https://www.linkedin.com/in/eliorboukhobza/>
[image: instagram] <https://www.instagram.com/heliosaian/>
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

On 5 January 2021 at 15:32:51, Michael Dolce (notifications@github.com)
wrote:

Yes, I have the latest version of Slack already (4.12.0). And I have
reinstalled it as you suggested. I am still seeing the same error

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/mallowigi/mtslack/issues/57#issuecomment-754638523>, or
unsubscribe

**Comment by mdolce8** [comment]
Sure I can join the slack channel, but I am having trouble finding it. Is there a link to it somewhere? 

Cool, thanks. And my computer is definitely 64bit.

**Comment by mallowigi** [comment]
https://join.slack.com/t/material-theme-ui/shared_invite/zt-4w78iblt-cHXYRYERpuGNbD~TqsD8rg

**Comment by mdolce8** [comment]
I solved my issue. I had previously installed Slack from homebrew and the Slack website. Neither of those worked. Instead I should have installed Slack from the Apple's App Store. Now Slack Tweaks work!

**Comment by MuathIbnHassan** [comment]
the problem happened to me with the version 4.12.2
maybe @mdolce8 thinks it solved because he went back to 4.12.0 in app store

I solved the problem by changing the line `slack-themes/lib/consts.js:9` from :
```javascript
const ASAR = `app-${ARCH}.asar`
```
to :
```javascript
const ASAR = `app.asar`
```

and i can confirm it's working with `npm run apply`
